c-sharp-razor/src/Minitwit.Infrastructure/Repository/CheepCreateValidator.cs

```diff
@@ -0,0 +1,17 @@
+using FluentValidation;
+using Minitwit.Core.Entities;
+
+namespace Minitwit.Infrastructure.Repository;
+
+public class MessageCreateValidator : AbstractValidator<CreateMessage>
+{
+    public MessageCreateValidator()
+    {
+        // @TODO Check that these values are correct:
+        RuleFor(x => x.Text)
+            .NotEmpty()
+            .MaximumLength(160)
+            .MinimumLength(5)
+            .WithMessage("The Message must be between 5 and 160 characters.(MessageCreateValidator)");
```

Comment:
This is a new requiremen, no?. We should discuss whether messages should be constrained to a certain limit :)

Reply:
https://www.postgresql.org/docs/current/datatype-character.html
I think @simoonH should remove it. Postgres text in theory can store any length (physically limited to 1GB).

Reply:
removed this